packages/default/test/renderers/slider.field.test.tsx

```diff
@@ -0,0 +1,476 @@
+    type: 'Control',
+    scope: {
+      $ref: '#/properties/foo'
+    }
+  };
+  t.is(
+    sliderFieldTester(
+      control,
+      {
+        type: 'object',
+        properties: {
+          foo: {
+            type: 'integer',
+            maximum: 10,
+            minimum: 2
+          }
+        }
+      }
+    ),
+    4
+  );
+});
+
+test('render', t => {
+  const schema: JsonSchema = {
+    type: 'object',
+    properties: {
+      foo: {
+        type: 'number',
+        maximum: 10,
+        minimum: 2
+      }
+    }
+  };
+  const store = initJsonFormsStore({ 'foo': 5 }, schema, t.context.uischema);
+  const tree = renderIntoDocument(
+    <Provider store={store}>
+      <SliderField schema={schema} uischema={t.context.uischema}/>
+    </Provider>
+  );
+
+  const input = findRenderedDOMElementWithTag(tree, 'input') as HTMLInputElement;
+  t.is(input.type, 'range');
+  t.is(input.value, '5');
+});
+
+test('update via input event', t => {
+  const store = initJsonFormsStore(t.context.data, t.context.schema, t.context.uischema);
+  const tree = renderIntoDocument(
+    <Provider store={store}>
+      <SliderField schema={t.context.schema} uischema={t.context.uischema}/>
+    </Provider>
+  );
+  const input = findRenderedDOMElementWithTag(tree, 'input') as HTMLInputElement;
+  input.value = '3';
+  change(input);
+  t.is(getData(store.getState()).foo, 3);
+});
+
+test('update via action', t => {
+  const store = initJsonFormsStore({ 'foo': 3 }, t.context.schema, t.context.uischema);
+  const tree = renderIntoDocument(
+    <Provider store={store}>
+      <SliderField schema={t.context.schema} uischema={t.context.uischema}/>
+    </Provider>
+  );
+  const input = findRenderedDOMElementWithTag(tree, 'input') as HTMLInputElement;
+  t.is(input.value, '3');
+  store.dispatch(update('foo', () => 4));
+  setTimeout(() => t.is(input.value, 'Bar'), 4);
+});
+
+test.failing('update with undefined value', t => {
+  const store = initJsonFormsStore(t.context.data, t.context.schema, t.context.uischema);
+  const tree = renderIntoDocument(
+    <Provider store={store}>
+      <SliderField schema={t.context.schema} uischema={t.context.uischema}/>
+    </Provider>
+  );
+  const input = findRenderedDOMElementWithTag(tree, 'input') as HTMLInputElement;
+  store.dispatch(update('foo', () => undefined));
+  t.is(input.value, '');
+});
+// FIXME this moves the slider and changes the value
+test.failing('update with null value', t => {
+  const store = initJsonFormsStore(t.context.data, t.context.schema, t.context.uischema);
+  const tree = renderIntoDocument(
+    <Provider store={store}>
+      <SliderField schema={t.context.schema} uischema={t.context.uischema}/>
+    </Provider>
+  );
+  const input = findRenderedDOMElementWithTag(tree, 'input') as HTMLInputElement;
+  store.dispatch(update('foo', () => null));
+  t.is(input.value, '');
+});
+// FIXME this moves the slider and changes the value
+test('update with wrong ref', t => {
```

Comment:
Is the comment accurate? Should this fail?

Reply:
I placed the FIXME at the wrong place, will fix it with the rebase